### DISCUSSION TITLE
How do VETO NFTs solve corruption?

### DISCUSSION THREAD

**Discussion by connormcmk** [RadicalxChange/salsa-nft#10] [Q&A]
In [this blog post](https://www.radicalxchange.org/media/blog/a-speculative-sketch-of-a-dao-with-open-collective/) it was offered that the purpose of the VETO NFT is to address the possibility of corruption.

> These are good things. But there’s an obvious objection to using SALSA NFTs for governance, which will already have some of you squirming in your seat: corruption. People might, for example, try to buy a bunch of Governance NFTs all at once, obtaining lots of temporary influence, and then pass extractive, self-interested governance measures. That would be very bad! And that’s where the Veto Tokens come in.

Here, corruption is defined as the ability for some nefarious actor to purchase a bunch of governance control quickly and then use that to do something bad.

This is used to motivate the next section where VETO NFTs are introduced. But it's not clear to me how VETO NFTs are in any way desirable:

1. VETO NFTs are still "corruptable": you can pay the holder to use it in the way that you want them to. 
2. VETO NFTs have all the same bad plutocratic attributes of a normal voting system (they don't cost you anything to own so you don't allocate their ownership to those that care)

Furthermore, to call these "veto rights" is really disingenuous. 
> And the QV tokens used in veto votes can be used for or against vetos.

This means that really what they represent is additional voting power allocated to a select group of people which can't be taken away. It's fine if you want to do this in your governance system, but it's not entirely clear why this isn't just a "divine rights of kings" sort of choice. It seems like giving inalienable governance control to a specially selected group of people is what we're trying to minimize, not codify.

Finally, it's worth thinking about what kind of "corruption" we're worried about with someone taking over the network. Let's say there was a DAO and it had a very important vote coming down the pipe and "nefarious" actor 123 decided to buy up sufficient governance tokens to dictate the outcome she wanted. And let's assume she succeeds. What would the community do?

Well they would say "thank you" to nefarious actor 123 for the money and fork the DAO. Now, the community has cash, the governance tokens are worth less (since they control something that matters less) and yet the community can still collaborate, just now in a new place.

In other words, if you maintain the property of forkability wouldn't you be safe to drop the VETO NFTs without worrying about corruption?

**Comment by connormcmk** [comment]
@alexrandaccio or @mortoni is this discussion forum the right place for this sort of question?
